src/scripts/pbs_submit.sh

```diff
@@ -132,7 +135,7 @@ bls_set_up_local_and_extra_args
 [ -z "$bls_opt_queue" ] || grep -q "^#PBS -q" $bls_tmp_file || echo "#PBS -q $bls_opt_queue" >> $bls_tmp_file
 
 # Extended support for MPI attributes
-if [ "x$bls_opt_wholenodes" == "xyes" ] ; then
+if [ "x$bls_opt_wholenodes" == "xyes" ] && [ "$is_pbs_pro" -neq "0" ]; then
```

Comment:
I'd like to get at least basic PBS Pro support into the blahp for the upcoming OSG release so can we just accept this PR? MPI support will take a better understanding of the PBS Pro MPI options than I currently have so it will be a while.